### DISCUSSION TITLE
Feature request: OAI-PMH interface

### DISCUSSION THREAD

**Discussion by wetneb** [knowledgefutures/pubpub#1373] [Ideas]
This project looks great, thanks for making it open source!

I think it would be useful to provide an OAI-PMH interface to the publications you store:
https://www.openarchives.org/OAI/openarchivesprotocol.html
This would make you eligible for inclusion in aggregation systems such as BASE:
https://www.base-search.net/

You probably also want to add `<meta>` tags to help Google Scholar index your content.
https://scholar.google.com/intl/en/scholar/inclusion.html

Related: issue #80

**Comment by gabestein** [comment]
Hi everyone — I'm going in and converting some of the longer-lived feature request/idea issues from Issues to Discussion threads. There's no exact heuristic about what belongs in the forum vs. what belongs as an issue, so **when in doubt, please post wherever you see fit — we won't mind!** But in general, we'd like to make Issues a space for things that we're committed to building/fixing, and Discussions a space for most everything else. So, if you're reporting a bug or requesting we make a straightforward change to an existing feature, go ahead and log an Issue. Otherwise, Discussions is likely a better bet.

So, in this case, we're moving it to discussions and inviting anyone who's also interested in the idea to comment!

(And in this case, we've fortunately already added lots of good meta tags for Google Scholar and others).
